libc/test.yaml

```diff
@@ -62,3 +62,14 @@ test:
       execute: test-libc-time
       targets:
         include: [host-generic-pc]
+
+    - name: unix-socket
+      execute: test-libc-unix-socket
+      targets:
+        exclude: [armv7m4-stm32l4x6-nucleo]
```

Comment:
Good point, but I think it will be better to fix this issue (if there is some) and then enable back the whole test or if the stm is just slower enable this test as nightly one.